babel/util.py

```diff
@@ -134,6 +134,16 @@ def parse_future_flags(fp: IO[bytes], encoding: str = 'latin-1') -> int:
     return flags
 
 
+def remove_trailing_zeros_after_decimal(string: str, decimal_symbol: str) -> str:
+    parts = string.split(decimal_symbol)
```

Comment:
This looks a bit suspicious, if `string` has more than one `decimal_symbol`. Should this use `partition` or `rpartition` instead, à la

```suggestion
    integer_part, _, decimal_part = string.partition(decimal_symbol)
```

Reply:
Thanks for the suggestion. I've used `partition` to ensure robustness. While there are already checks that should prevent multiple `decimal_symbol` at that stage of the function, I agree that using `partition` adds a layer of safety and better communicates what the function is trying to do.